Mathlib/GroupTheory/GroupAction/Blocks.lean

```diff
@@ -38,8 +38,9 @@ The non-existence of nontrivial blocks is the definition of primitive actions.
 
 - `MulAction.BlockMem` : the type of blocks containing a given element
 
-- `MulAction.BlockMem.boundedOrder` :
-  the type of blocks containing a given element is a bounded order.
+## instance
+
+- The type of blocks containing a given element is a bounded order.
```

Comment:
I still don't understand this change. The name is wrong, I agree, but I don't think instances should necessarily be listed separately
```suggestion
- `MulAction.BlockMem.instBoundedOrder` :
  the type of blocks containing a given element is a bounded order.
```